### DISCUSSION TITLE
How to do a dependent validation in RHF

### DISCUSSION THREAD

**Discussion by abish373** [react-hook-form/react-hook-form#7328] [Q&A] [answered]
I'm trying to do a dependent validation. Using `deps` in register is not working for me. [Please take a look at the issue here...](https://stackoverflow.com/questions/70380730/how-to-clear-react-hook-form-dependent-field-errors/70380922)

**Comment by bluebill1049** [comment, marked as answer]
not sure if this works for you: 
https://codesandbox.io/s/rhf-interactive-field-forked-3pnun?file=/src/App.js

**Reply by abish373** [reply]
Thanks for your reply @bluebill1049 . When user increased desired field and increasing the max field shows error

![image](https://user-images.githubusercontent.com/53331748/146927210-5e558ae6-7652-46b6-b48f-101e34b4ae59.png)

**Reply by bluebill1049** [reply]
updated to the latest version, your original csb was an old version. check it out again

**Reply by abish373** [reply]
Thank you very much, it works wonderfully
